flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/iomanager/IOManagerITCase.java

```diff
@@ -189,15 +189,13 @@ public void parallelChannelsTest() throws Exception {
                         try {
                             intValue = Integer.parseInt(val.value);
                         } catch (NumberFormatException nfex) {
-                            Assert.fail(
-                                    "Invalid value read from reader. Valid decimal number expected.");
+                            fail("Invalid value read from reader. Valid decimal number expected.");
```

Comment:
fail with catch can be updated to assertThatThrownBy

Reply:
I think we cannot use `assertThatThrownBy ` here.
It seems that this exception should not occur in this case, maybe we should use `assertThatNoException` here.
Shall we make that change?

Reply:
Sorry, it's my mistake. 

We should leave as it is, the `fail` can provide the friend information to troubleshooting.